metadata-ingestion/src/datahub/ingestion/source/unity/proxy.py

```diff
@@ -134,6 +134,13 @@ def check_connectivity(self) -> bool:
         self._unity_catalog_api.list_metastores()
         return True
 
+    def assigned_metastore(self) -> Optional[Metastore]:
+        response: dict = self._unity_catalog_api.get_metastore_summary()
+        if response.get("metastore_id") is None:
+            logger.info("Not found assinged metastore")
```

Comment:
Typo on `assigned`. Also, "Assigned metastore not found" would make a little more sense to me, but I don't feel strongly either way

Reply:
+1